week1/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>help hamsick</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="headerColorBox">
+      <h1>💛🍔 햄식이를 도와줘 🍔💛</h1>
+      </div>
+    </header>
+
+    <main>
+      <aside class="sidebar">
+        <section class="option_card">
+          <h2 class="optionTitle">다른거 먹자</h2>
+          <div class="option"> 
+            <p class="macdonalds">맥도날드</p>
+            <p class="burgerking">버거킹</p>
+          </div>
+        </section>
+      </aside>
```

Comment:
```suggestion
      <aside class="sidebar">
          <h2 class="sidebar__title">다른거 먹자</h2>
          <ul class="sidebar__option"> 
            <li class="sidebar__option-li">맥도날드</li>
            <li class="sidebar__option-li">버거킹</li>
          </ul>
      </aside>
```
1. aside테그와 section테그가 모두 구획을 나눠주는 테그라 둘 중 하나만 사용해서 스타일링 하는 편이 좀 더 시멘틱한 것 같아요! aside테그 자체를 박스로 만들고, 이 박스의 배치에 대한 스타일을 상위 테그인 main에서 하는 편이 좀 더 직관적일 것 같습니다!
2. option은 list테그를 사용하는 편이 좀 더 시멘틱할 것 같아요! 아마 기본적으로 list에 먹혀있는 css스타일이 있을텐데, 테그를 시맨틱하게 사용하기 위해서 처음에 모든 테그에 대한 기본 css 를 reset시켜주는 reset.css를 링크해두면 아마 스타일링 하기 더 편리할 것 같습니다!  위에 어떤 식으로 링크할 수 있는지 적어둘게요 ! 
3. BEM방식으로 어떻게 클래스 네임을 짓는지 느낌을 알았으면 좋겠어서 클레스네임도 한 번 바꿔봤습니다 ! 참고하면 좋을 것 같습미다

Reply:
아 그렇군요 구획을 나누는 태그라는 거 쇽샥 알아갑니다 :) 그리고, 질문이있어요! 제가 버거킹이랑 맥도날드 css에서 색을 다르게 적용해두었는데, 저렇게 classname을 같은걸로 적용한 후에 다른 색을 설정해야하는 경우에는 :nth-child(num) 요걸로 css 추가해주면 될까요? 아니면 다른 방법이 있을까요?!

Reply:
@songah119 이건 컨벤션의 영역일 것 같아요!  :nth-child(num)로 스타일을 줘도 좋을 것 같고, 클래스명을 달리줘도 될 것 같습니다! 근데 리스트 요소에 대해서는  :nth-child(num)를 사용하는 것이 조금 더 일반적인 것 같아요 !